UI/ChatScreen.cpp

```diff
@@ -0,0 +1,204 @@
+	//freeze  the ui when using ctrl + C hotkey need workaround
+	if (g_Config.bBypassOSKWithKeyboard && !g_Config.bFullScreen)
+	{
+		std::wstring titleText = ConvertUTF8ToWString(n->T("Chat"));
+		std::wstring defaultText = ConvertUTF8ToWString(n->T("Chat Here"));
+		std::wstring inputChars;
+		if (System_InputBoxGetWString(titleText.c_str(), defaultText, inputChars)) {
+			//chatEdit_->SetText(ConvertWStringToUTF8(inputChars));
+			sendChat(ConvertWStringToUTF8(inputChars));
+		}
+	}
+#endif
+	chatEdit_->OnEnter.Handle(this, &ChatMenu::OnSubmit);
+	bottom->Add(new Button(n->T("Send")))->OnClick.Handle(this, &ChatMenu::OnSubmit);
+#elif defined(__ANDROID__)
+	bottom->Add(new Button(n->T("Chat Here"),new LayoutParams(FILL_PARENT, WRAP_CONTENT)))->OnClick.Handle(this, &ChatMenu::OnSubmit);
+#endif
+	chatVert_ = scroll_->Add(new LinearLayout(ORIENT_VERTICAL, new LinearLayoutParams(FILL_PARENT, WRAP_CONTENT)));
+	chatVert_->SetSpacing(0);
+	parent->Add(outer);
+}
+
+void ChatMenu::CreateViews() {
+	using namespace UI;
+
+	I18NCategory *n = GetI18NCategory("Networking");
+	UIContext &dc = *screenManager()->getUIContext();
+
+	AnchorLayout *anchor = new AnchorLayout(new LayoutParams(FILL_PARENT, FILL_PARENT));
+	anchor->Overflow(false);
+	root_ = anchor;
+
+	float yres = screenManager()->getUIContext()->GetBounds().h;
+
+	switch (g_Config.iChatScreenPosition) {
+	case 0:
+		box_ = new LinearLayout(ORIENT_VERTICAL, new AnchorLayoutParams(PopupWidth(), FillVertical() ? yres - 30 : WRAP_CONTENT, 280, NONE, NONE, 250, true));
+		break;
+	case 1:
+		box_ = new LinearLayout(ORIENT_VERTICAL, new AnchorLayoutParams(PopupWidth(), FillVertical() ? yres - 30 : WRAP_CONTENT, dc.GetBounds().centerX(), NONE, NONE, 250, true));
+		break;
+	case 2:
+		box_ = new LinearLayout(ORIENT_VERTICAL, new AnchorLayoutParams(PopupWidth(), FillVertical() ? yres - 30 : WRAP_CONTENT, NONE, NONE, 280, 250, true));
+		break;
+	case 3:
+		box_ = new LinearLayout(ORIENT_VERTICAL, new AnchorLayoutParams(PopupWidth(), FillVertical() ? yres - 30 : WRAP_CONTENT, 280, 250, NONE, NONE, true));
+		break;
+	case 4:
+		box_ = new LinearLayout(ORIENT_VERTICAL, new AnchorLayoutParams(PopupWidth(), FillVertical() ? yres - 30 : WRAP_CONTENT, dc.GetBounds().centerX(), 250, NONE, NONE, true));
+		break;
+	case 5:
+		box_ = new LinearLayout(ORIENT_VERTICAL, new AnchorLayoutParams(PopupWidth(), FillVertical() ? yres - 30 : WRAP_CONTENT, NONE, 250, 280, NONE, true));
+		break;
+	}
+
+	root_->Add(box_);
+	box_->SetBG(UI::Drawable(0x66303030));
+	box_->SetHasDropShadow(false);
+
+	View *title = new PopupHeader(n->T("Chat"));
+	box_->Add(title);
+
+	CreatePopupContents(box_);
+#if defined(_WIN32) || defined(USING_QT_UI)
+	//not work yet for tywald requests cant set the focus to chat edit after chat opened
+	root_->SetDefaultFocusView(box_);
+	box_->SubviewFocused(chatEdit_);
+	root_->SetFocus();
+#else
+	root_->SetDefaultFocusView(box_);
+#endif
+	UpdateChat();
+	chatScreenVisible = true;
+	newChat = 0;
+}
```

Comment:
I tested a little and found a solution. 
	`UI::EnableFocusMovement(true);`
When I inserted this code into the last line of CreateViews(), focusing correctly.
I don't know this way is correct or not but it can fix the issue anyway.